big_scape/data/sqlite.py

```diff
@@ -391,7 +391,7 @@ def init_run(run: dict) -> None:
             .values(
                 label=run["label"],
                 start_time=str(run["start_time"])[:-7],
-                mode=run["mode"],
+                mode=str(run["mode"]),
```

Comment:
```suggestion
                mode=run["mode"].name,
```
`str` turns into e.g. `RUN_MODE.CLUSTER`; `.name` will result in `CLUSTER`.